com/win32com/src/PyRecord.cpp

```diff
@@ -531,7 +708,13 @@ PyObject *PyRecord::getattro(PyObject *self, PyObject *obname)
         // in the last parameter, i.e. 'sub_data == NULL'.
         this_data = (BYTE *)psa->pvData;
         for (i = 0; i < nelems; i++) {
-            PyTuple_SET_ITEM(ret_tuple, i, new PyRecord(sub, this_data, pyrec->owner));
+            PyRecord *rec = PyRecord::new_record(sub, this_data, pyrec->owner);
+            if (rec == NULL) {
+                Py_DECREF(ret_tuple);
+                ret_tuple = NULL;
+                goto array_end;
+            }
+            PyTuple_SET_ITEM(ret_tuple, i, rec);
```

Comment:
oops, now I look again, this shouldn't even be a tuple 😅 . No need to change that now though.